docs/development/build-instructions-osx.md

```diff
@@ -71,7 +71,14 @@ to build Electron so there is no Xcode project generated.
 
 ```sh
 $ cd electron
-$ ./script/bootstrap.py -v
+$ npm run bootstrap
+```
+
+You can also bootstrap for debug builds:
+
+```sh
+$ cd electron
+$ npm run bootstrap:dev
```

Comment:
Same suggestions as above for better explanation of dev and non-dev meaning, and also misuse of `debug` here